packages/makeswift/src/utils/get-name.ts

```diff
@@ -79,7 +85,7 @@ async function askApprovalToIntegrateIfNeeded(dir: string): Promise<void> {
     const approval = await askApproval()
 
     if (!approval) {
-      throw new MakeswiftError('Will not integrate Next app.')
+      throw new MakeswiftError()
```

Comment:
This is an interesting way to abort. Could this not just be a `return`?

Reply:
This is a mistake, but I think return would be too (it's pretty deep in the stack!)

I've gone with `process.exit(0)`

Reply:
Ok, cool. Are there any issues with `process.exit(0)` not behaving like a `throw` or `return`? As in, do we risk some code running after the stack of this function unwinds while Node.js is attempting to exit, or does it immediately stop executing when `process.exit` is called?

Reply:
I don't think we'll have any issues. https://stackoverflow.com/questions/37818846/what-happens-when-process-exit-used-in-nodejs-program

I'm not certain no other lines of code will be run, but just from quick testing it was fine.

We'd probably have more problems if we had background async stuff happening.

Reply:
Ok, cool. I tested these two scripts and they run as expected:

```js
process.addListener('exit', () => {
  console.log('Exiting...')
})
process.exit(0)
```

```
process.exit(0)
console.log('Unreachable.')
```